v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/options/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 21 Google Inc.
```

Comment:
License header should be similar to the existing files with full year format (i.e Copyright (C) 2021 Google Inc.)